ivy/functional/backends/torch/experimental/manipulation.py

```diff
@@ -380,3 +381,6 @@ def trim_zeros(
     start = torch.argmin(nz) if "f" in trim.lower() else 0
     end = torch.argmin(torch.flip(nz, (0,))) if "b" in trim.lower() else 0
     return filt[start : len(filt) - end]
+
+
+trim_zeros.support_native_out = False
```

Comment:
Not sure why this gets added everywhere in the codebase, but you don't have to specify `support_native_out=False` as it always default to `False`, and you only need to add this if it's `True`

Reply:
Should I remove all `support_native_out=False` while I'm at it?

Reply:
Got it, I have removed it.